allure-junit5/src/test/java/io/qameta/allure/examples/junit5/AllureTitleTest.java

```diff
@@ -0,0 +1,21 @@
+package io.qameta.allure.examples.junit5;
+
+import io.qameta.allure.Allure;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class AllureTitleTest {
+
+    @Test
+    @DisplayName("Static name")
```

Comment:
this annotatioin can be applied to the test class too – so probably we have to reflect this usage scenario too...